src/main/java/org/brapi/test/BrAPITestServer/service/pheno/ScaleService.java

```diff
@@ -119,7 +119,7 @@ public ScaleEntity updateEntity(ScaleEntity entity, ScaleBaseClass scale) throws
 
 		ontologyService.updateOntologyReference(entity, scale.getOntologyReference());
 
-		if (scale.getValidValues() != null) {
+		if (scale.getValidValues() != null && scale.getValidValues().isPresent()) {
```

Comment:
scale.getValidValues().isPresent() seems redundant as it is checked on the next line

Reply:
Good catch, consolidated the redundant logic